### PR TITLE
Update code samples

### PR DESCRIPTION
**Pull request by jorgebay**
Use a package.json with dependencies for code sample modules.

### CODE REVIEW

**Comment by tolbertam** [comment]
Added CI enablement ([NODEJS-282](https://datastax-oss.atlassian.net/browse/NODEJS-282)) and also updated runner to time out runs (for [NODEJS-283](https://datastax-oss.atlassian.net/browse/NODEJS-283)) and count number of failures.

File: build.yaml [hunk 1]
```diff
@@ -24,4 +24,14 @@ build:
   - npm: install
   - npm: run-script ci
     graceful: true
+  - script: |
+      cd examples
+      npm install
+      npm install ../
```
Comment (jorgebay) [inline build.yaml]: Is this needed?

Comment (tolbertam) [inline build.yaml]: The `npm install ../` step points to the local cassandra driver, where as if i were to just use npm install, it'd use the version from master.

Comment (jorgebay) [inline build.yaml]: You are right! I've changed the dependency to point to local file in the package.json:
```
"dependencies": {
  "cassandra-driver": "file:../",
  // ...
}
```

It should not be required now.

Comment (tolbertam) [inline build.yaml]: Nice 👍

**Comment by tolbertam** [comment]
lgtm 👍

**Pull request merged**